readthedocs/notifications/backends.py

```diff
@@ -0,0 +1,78 @@
+"""Notification to message backends"""
+
+from django.conf import settings
+from django.contrib.messages import add_message
+from django.utils.module_loading import import_string
+from messages_extends.constants import INFO_PERSISTENT
+
+from readthedocs.core.utils import send_email
+
+from .constants import LEVEL_MAPPING, REQUIREMENT, HTML
+
+
+def send_notification(request, notification):
+    """Send notifications through all backends defined by settings
+
+    Backends should be listed in the settings ``NOTIFICATION_BACKENDS``, which
+    should be a list of class paths to be loaded, using the standard Django
+    string module loader.
+    """
+    backends = getattr(settings, 'NOTIFICATION_BACKENDS', [])
+    for cls_name in backends:
+        backend = import_string(cls_name)(request)
+        backend.send(notification)
```

Comment:
Is there a way to send a message to only one of the backends? Is that something we want?